src/features/Layout/Navigation/components/NavTray.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { Box, Icon, Text } from '@components';
+import { BREAK_POINTS, COLORS } from '@theme';
+import { INavTray } from '@types';
+
+import { Tray } from './Tray';
+
+const SBox = styled(Box)`
+  cursor: pointer;
+`;
+
+const SText = styled(Text)`
+  text-transform: uppercase;
+`;
```

Comment:
Can we use the `sx` prop for simple changes like these?
I avoid the extra declaration

Reply:
We can't use `sx` in `styled-system` it's something from `rebass` :(

Reply:
I'll use `style` instead